cloud/gcpprovider.go

```diff
@@ -435,6 +520,11 @@ func (gcp *gcpKey) Features() string {
 	} else {
 		usageType = "ondemand"
 	}
+
+	if _, ok := gcp.Labels["cloud.google.com/gke-accelerator"]; ok {
```

Comment:
Making this a constant would likely go a long way toward avoiding updated-in-one-place-but-not-the-other bugs

Reply:
Done